_alerts/2017-07-12-disruptive-forks.md

```diff
@@ -0,0 +1,86 @@
+---
+## This file is licensed under the MIT License (MIT) available on
+## http://opensource.org/licenses/MIT.
+
+title: "Potential disruptive chain forks"
+shorturl: "disruptive-forks"
+active: true
+banner: "Warning: Bitcoin may be unsafe to use on August 1st (click here for details)"
```

Comment:
I think the current title is better because it describes the consequence in plain English rather than the consequence in technical jargon.

For the parenthetical, we've done that with previous alerts because it isn't clear that the banner can be clicked.  E.g.:

```text
user@devbco:~/alerts/_alerts$ git grep 'click here'
2015-07-04-spv-mining.md:#banner: "WARNING: many wallets currently vulnerable to double-spending of confirmed transactions (click here to read)"
2015-10-12-upnp-vulnerability.md:#banner: "WARNING: serious vulnerability in UPnP library used by Bitcoin Core (click here to read)"
2016-11-01-alert-retirement.md:## banner: "Alert system is being retired (click here to read)"
2017-07-12-disruptive-forks.md:banner: "Warning: Bitcoin may be unsafe to use on August 1st (click here for details)"
```

I continue to think that it's a good idea.